manim/scene/scene.py

```diff
@@ -72,6 +74,9 @@ def __init__(self, **kwargs):
         except EndSceneEarlyException:
             pass
         self.tear_down()
+        # We have to reset these settings in case of multiple renders.
+        file_writer_config["skip_animations"] = False
```

Comment:
Suggestion: perhaps we should log the fact that the config is being changed.

Reply:
I don't think its necessay, as skip_animations is in theory never used by the user. 
Tell me if you disagree on this point.